src/spikeinterface/core/binaryrecordingextractor.py

```diff
@@ -55,20 +56,26 @@ def __init__(
         self,
         file_paths,
         sampling_frequency,
-        num_chan,
         dtype,
+        num_channels=None,
         t_starts=None,
         channel_ids=None,
         time_axis=0,
         file_offset=0,
         gain_to_uV=None,
         offset_to_uV=None,
         is_filtered=None,
+        num_chan=None,
     ):
+        num_channels = num_channels or num_chan
```

Comment:
What if they are both given? and maybe different? e.g. `num_channels=16` and `num_chan=32`?

Reply:
This:
https://en.wikipedia.org/wiki/Short-circuit_evaluation

Together with this:
https://docs.python.org/3/library/stdtypes.html#truth-value-testing

It is a common python idiom but maybe I should make the point of avoiding it as it can be confusing.